package.json

```diff
@@ -64,6 +65,7 @@
     "iron": "4.0.5",
     "jest": "23.1.0",
     "joi": "13.4.0",
+    "js-joda": "^1.8.0",
```

Comment:
This is an interesting case. Currently `meetup-web-components` has several peerDependencies that cause `yarn install` warnings in `mwp-*` packages because they are missing. Example:

```
warning " > meetup-web-components@5.2.2434" has unmet peer dependency "classnames@^2.2.5".
warning " > meetup-web-components@5.2.2434" has unmet peer dependency "js-joda@>1.8.0".
warning " > meetup-web-components@5.2.2434" has unmet peer dependency "swarm-animation@^0.0.95".
warning " > meetup-web-components@5.2.2434" has incorrect peer dependency "swarm-icons@^3.4.334".
warning " > meetup-web-components@5.2.2434" has unmet peer dependency "swarm-sasstools@^5.0.412 || ^4.2.408".
```
There are two options I can see in handling this:
1) The option shown in this PR -- add all the peer dependencies to `package.json` files of all `mwp-*` packages that depend on `meetup-web-components`.
2) Another option is to remove the peerDependences in `meetup-web-components` and make them dependencies. However i'm not familiar with the decision as the why they are peerDependencies to begin with.

Reply:
peer dependencies help avoid having multiple versions of a particular package built into the app - e.g. if MWP uses js-joda and specifies v1.8.0 as a dependency, and then some mup-web code wants to use js-joda and so installs version 1.8.2, _both_ versions will be bundled by webpack into the client script.

Specifying a peerDependency is the same as saying "I will use whatever you have installed that matches this version range" - same thing for React and other packages that get directly imported in the platform and in the apps where we want to avoid double-bundling.

Reply:
that said, we might want to find a way to reduce the peer dependencies for MWC... there are a lot

Reply:
Interesting. The versioning scenario makes sense.

I always read peerDependencies are requirements as well. Is that not the case? Or do we not want that to be the case?

Reply:
they're requirements in order to avoid the double-bundling issue, but it's always a judgement call based on

1. how likely is it that the consumer will need to use the same dependency itself?
   - not likely: define it as a direct dependency
   - very likely: probably good to make it a peer dependency so that it will always be shared
2. how big is the dependency?
   - small: not a major problem to double-bundle
   - large: make every effort to avoid double-bundling

So, 'peerDependencies' is a "guaranteed-safe' way of sharing a dependency, but it creates some logistical/maintenance burden... I don't think there's a good bright-line test on whether/when to use it